src/main/java/br/ufpe/cin/mergers/handlers/renaming/SafeRenamingHandler.java

```diff
@@ -57,6 +73,17 @@ private void handleSingleRenaming(MergeContext context, Quartet<FSTNode, FSTNode
         RenamingUtils.runTextualMerge(context, leftNode, baseNode, rightNode, mergeNode);
     }
 
+    /**
+     * When both developers renamed or deleted a method, we classify the renaming as mutual.
+     * Then, we run a decision tree based on which renaming type each developer did.
+     * 
+     * For example, if both developers renamed without body changes, we check if they renamed
```

Comment:
should we list all cases of the decision tree instead of just a "for example"?
@pauloborba  @jvcoutinho

Reply:
add at least a link for the table

Reply:
@jvcoutinho  you could put it here https://github.com/guilhermejccavalcanti/jFSTMerge/tree/master/documentation